meta/saisanitycheck.c

```diff
@@ -332,6 +332,26 @@ bool sai_metadata_is_acl_field_or_action(
     return false;
 }
 
+bool sai_metadata_is_tlv_or_segment(
```

Comment:
@kcudnik - I was having trouble with this and used this as a workaround.  Since sai_tlv_list_t and sai_segment_list_t are structures defined in saitypes.h, I couldn't find an example of a default value of a struct.  Most examples are enums or pointers.  I tried 0, NULL without success.  Other examples like qos_map_list_t / tunnel_map_list_t doesn't have a default value defined.

If you can point me out an example, that would help me a lot.